tests/virt/node/general/test_vmi_reset.py

```
@@ -0,0 +1,58 @@
+class TestVMIReset:
+    @pytest.mark.polarion("CNV-12372")
+    def test_reset_changes_boot_time(self, vm_for_reset_test, vm_initial_boot_time, vm_after_reset):
+        assert vm_after_reset > vm_initial_boot_time, (
+            f"Boot time did not change after reset. Initial: {vm_initial_boot_time}, Current: {vm_after_reset}"
+        )
+
+        current_launcher_pod_uid = vm_for_reset_test.vmi.virt_launcher_pod.instance.metadata.uid
+        assert current_launcher_pod_uid, "Launcher pod UID should be present after reset"
+
+    @pytest.mark.polarion("CNV-12373")
+    def test_reset_preserves_ephemeral_disk_state(self, vm_for_reset_test, vm_ephemeral_marker_creation):
```

Ruff (0.14.3)

Unused method argument: `vm_ephemeral_marker_creation`

(ARG002)